CreateModels/qSlicerCreateModelsModule.cxx

```diff
@@ -65,14 +65,14 @@ qSlicerCreateModelsModule::~qSlicerCreateModelsModule()
 //-----------------------------------------------------------------------------
 QString qSlicerCreateModelsModule::helpText()const
 {
-  return "Module for creating simple shapes, such as sphere, cylinder, prism, needle, and coordinate system axes."
-    "For help on how to use this module visit: <a href='http://www.slicerigt.org'>SlicerIGT website</a>.";
+  return qSlicerCreateModelsModule::tr("Module for creating simple shapes, such as sphere, cylinder, prism, needle, and coordinate system axes."

```

Comment:
`tr` should be sufficient, except in special cases. Please read https://github.com/Slicer/SlicerLanguagePacks/blob/main/DevelopersManual.md thoroughly and review all similar cases.